src/components/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import GlobalStyles from '../styles/GlobalStyles';
+import Entry from './Entry';
+import { theme } from '../styles/theme';
+
+function App() {
+  return (
+    <ThemeProvider theme={theme}>
+      <>
+        <GlobalStyles />
+        <Entry title="Oliver Smith Show" amount="37.50" />
```

Comment:
opt* when you pass a prop in quotes like `amount="35.75" it automatically becomes a string, regardless of what the actual value is. It's fine for the purpose of this issue, but if you needed the number in the component for something like math, it wouldnt work without having to parse it. If you want to pass the 'type' of variable, do it with brackets like `amount={35.76}`. You'll end up changing this prop later anyway so up to you if you wanna edit this now.